scripts/generate_madenearme_page.pl

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/perl -w
+
+# This file is part of Product Opener.
+#
+# Product Opener
+# Copyright (C) 2011-2019 Association Open Food Facts
+# Contact: contact@openfoodfacts.org
+# Address: 21 rue des Iles, 94100 Saint-Maur des Fossés, France
+#
+# Product Opener is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use Modern::Perl '2017';
+use utf8;
+
+use CGI::Carp qw(fatalsToBrowser);
+use CGI qw/:cgi :form escapeHTML/;
+
+use ProductOpener::Config qw/:all/;
+use ProductOpener::Store qw/:all/;
+use ProductOpener::Index qw/:all/;
+use ProductOpener::Display qw/:all/;
+use ProductOpener::Users qw/:all/;
+use ProductOpener::Products qw/:all/;
+use ProductOpener::Food qw/:all/;
+use ProductOpener::Tags qw/:all/;
+
+
+use CGI qw/:cgi :form escapeHTML/;
+use URI::Escape::XS;
+use Storable qw/dclone/;
+use Encode;
+use JSON::PP;
+use Log::Any qw($log);
+
+use ProductOpener::Lang qw/:all/;
+
+$cc = $ARGV[0];
+$lc = $ARGV[1];
+
+$lang = $lc;
+
+if ((not defined $cc) or (not defined $lc)) {
+	die("Pass country code (or world) and language code as arguments.\n");
+}
+else {
+		if (defined $country_codes{$cc}) {
+			$country = $country_codes{$cc};
+		}
+		else {
+			$country = "en:world";
```

Comment:
I think we should throw an error here and not just pretend that world is selected

Reply:
This line belonged to the other PR